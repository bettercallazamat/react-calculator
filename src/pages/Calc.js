import React from 'react';
import App from '../components/App';

const Calc = () => (
  <div>
    <h2 className="calcText">Lorem Ipsum</h2>
    <div className="calcApp">
      <App />
    </div>
  </div>
);

export default Calc;
