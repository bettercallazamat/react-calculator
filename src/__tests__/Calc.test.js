import React from 'react';
import { render } from '@testing-library/react';
import Calc from '../pages/Calc';

describe('rendered Home page', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Calc />);
    expect(asFragment(<Calc />)).toMatchSnapshot();
  });
});
