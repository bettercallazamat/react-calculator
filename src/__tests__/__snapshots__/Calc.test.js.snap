// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Home page should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="calcDiv"
  >
    <div
      class="calcText"
    >
      <h2>
        Nobody:
      </h2>
      <h2>
        Literally no one:
      </h2>
      <h2>
        YOU: Lets do some math!
      </h2>
    </div>
    <div
      class="calcApp"
    >
      <div
        class="App"
      >
        <div
          class="Display"
        >
          0
        </div>
        <div
          class="ButtonPanel"
        >
          <div
            class="btn-group btn-group1"
          >
            <button
              class="Button gray"
              type="button"
            >
              AC
            </button>
            <button
              class="Button gray"
              type="button"
            >
              +/-
            </button>
            <button
              class="Button gray"
              type="button"
            >
              %
            </button>
            <button
              class="Button orange"
              type="button"
            >
              รท
            </button>
          </div>
          <div
            class="btn-group btn-group2"
          >
            <button
              class="Button gray"
              type="button"
            >
              7
            </button>
            <button
              class="Button gray"
              type="button"
            >
              8
            </button>
            <button
              class="Button gray"
              type="button"
            >
              9
            </button>
            <button
              class="Button orange"
              type="button"
            >
              x
            </button>
          </div>
          <div
            class="btn-group btn-group3"
          >
            <button
              class="Button gray"
              type="button"
            >
              4
            </button>
            <button
              class="Button gray"
              type="button"
            >
              5
            </button>
            <button
              class="Button gray"
              type="button"
            >
              6
            </button>
            <button
              class="Button orange"
              type="button"
            >
              -
            </button>
          </div>
          <div
            class="btn-group btn-group4"
          >
            <button
              class="Button gray"
              type="button"
            >
              1
            </button>
            <button
              class="Button gray"
              type="button"
            >
              2
            </button>
            <button
              class="Button gray"
              type="button"
            >
              3
            </button>
            <button
              class="Button orange"
              type="button"
            >
              +
            </button>
          </div>
          <div
            class="btn-group btn-group5"
          >
            <button
              class="Button gray btnWide"
              type="button"
            >
              0
            </button>
            <button
              class="Button gray"
              type="button"
            >
              .
            </button>
            <button
              class="Button orange"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
